### PR TITLE
Apply rustfmt to all .rs files.

### PR DESCRIPTION
**Pull request by daschl**
This changeset is using the default ruleset of rustfmt
version 0.7.1.

Note that it doesn't change any functionality, just formatting changes.

### CODE REVIEW

**Comment by daschl** [comment]
Not sure while it fails, tensorflow compiles fine locally (looks like a tensorflow issue)

**Comment by adamcrume** [comment]
The build problem is that TensorFlow build times have increased enough to cause Travis to time out.  I haven't been able to get the build times down, but I think that soon we'll be able to pull in pre-built TensorFlow binaries.

I'll take a look at your changes this evening.

**Comment by daschl** [comment]
@adamcrume sure, no problem. Note that current master build works fine, although cargo test fails because an import is not working in one of the examples (but I reproduced this on master as well so not related to this PR I think :))

File: src/graph.rs [hunk 1]
```diff
@@ -471,380 +498,444 @@ type Port<'a> = Output<'a>;
...
-        c_attr_name.as_ptr(),
-        value.as_ptr() as *const c_void,
-        // Allow trivial_numeric_casts because usize is not necessarily size_t.
-        value.len() as size_t,
-        status.inner);
-    }
-    status.as_result()
-  }
-}
+
```
Comment (adamcrume) [inline src/graph.rs]: Please remove the redundant blank lines.

File: src/lib.rs [hunk 2]
```diff
@@ -81,17 +82,20 @@ macro_rules! impl_drop {
 ////////////////////////
 
 // We would like to use the pattern:
-//   ($doc:expr, $c_name:ident, $enum_name:ident { $( $(#[$attr:meta])* $name:ident = $num:expr),* })
+//   ($doc:expr, $c_name:ident, $enum_name:ident
+// { $( $(#[$attr:meta])* $name:ident = $num:expr),* })
```
Comment (adamcrume) [inline src/lib.rs]: Indent this past the parenthesis on the previous line.

File: src/lib.rs [hunk 3]
```diff
@@ -81,17 +82,20 @@ macro_rules! impl_drop {
...
 // so the enum variants would look like:
 //   /// Denotes a foo.
 //   Foo = 1,
 // but the compiler complains:
-//   error: local ambiguity: multiple parsing options: built-in NTs ident ('name') or 1 other option.
+//   error: local ambiguity: multiple parsing options: built-in NTs ident ('name')
+// or 1 other option.
```
Comment (adamcrume) [inline src/lib.rs]: Indent this line.

**Comment by daschl** [comment]
@adamcrume applied your comments and merged it with the old commit.

**Comment by adamcrume** [comment]
Awesome, thanks!

**Pull request merged**